ArticleTemplates/assets/js/modules/quiz.js

```diff
@@ -1,251 +1,454 @@
-                    var answerImg = answer.querySelectorAll(':scope > img');
-                    if (answerImg.length) {
-                        if ($(answerImg).attr('src') !== '') {
-                            $(answer).addClass('has-image');
-                            $(answerImg).addClass('answer__img');
-                            $(answerWrapper).append(answerImg);
-                        } else {
-                            $(answerImg).remove();
-                            answerImg = '';
-                        }
+            adjustText: function (parent, text) {
+                parent.appendChild(text);
+            },
+
+            IsAdBelowQuiz: function (quiz) {
+                var IsAdBelowQuiz = false,
+                    mpu = document.querySelector('.advert-slot__wrapper'),
+                    mpuOffset,
+                    quizOffset;
+
+                if (mpu) {
+                    mpuOffset = mpu.offsetTop;
+                    quizOffset = quiz.offsetTop;
+
+                    if (mpuOffset > quizOffset) {
+                        IsAdBelowQuiz = true;
                     }
+                }
+
+                return IsAdBelowQuiz;
+            },
+
+            adjustAdPosition: function (yPos, startTime, timeStamp) {
+                var newYPos,
+                    progress;
+
+                if (!startTime) {
+                    startTime = timeStamp;
+                }
+
+                progress = timeStamp - startTime;
+                newYPos = Ads.modules.updateMPUPosition(yPos);
+
+                if (progress < 2000) {
+                    window.animFrame(modules.adjustAdPosition.bind(null, newYPos, startTime));
+                }
+            },
+
+            onNewsAnswerClick: function (answer, question, isImage) {
+                var answerPara,
+                    correctAnswerWrapper,
+                    startTime = null,
+                    yPos = null;
+
+                if (question.classList.contains('answered')) {
+                    return;
+                }
+
+                if (answer.dataset.correct === "true") {
+                    question.classList.add('is-correct');                    
+                    modules.score++;
+                } else {
+                    answer.classList.add('wrong-answer');
+                    question.classList.add('is-wrong');
+                    answer = question.querySelector('[data-correct="true"]');
+                }
+
+                answer.classList.add('correct-answer');
+
+                if (answer.dataset.correctAnswerExplanation) {
+                    answerPara = document.createElement("p");
+                    answerPara.classList.add("answer__explanation");
+                    answerPara.innerHTML = answer.dataset.correctAnswerExplanation.trim();
+                    correctAnswerWrapper = answer.querySelector('.answer__message');
```

Comment:
Minor style thing (sorry, I know I would leave code style ;)), you're mixing single and double quotes for strings in this file